README.md

```diff
@@ -188,6 +188,44 @@ We can always use layer to search for data availability:
   const datesS1 = await layerS1.findDatesUTC(bbox, fromTime, toTime);
 ```
 
+
+## Cancelling requests
+
+You can also cancel requests when searching/fetching data.
+
+To do so a factory of tokens needs to be created and a token passed through a request configuration object. In this object can also be defined the amount of retires desired in the case the petition fails.
```

Comment:
- > ...a factory of tokens needs to be created...

  Technically, factory is not created, it is just used to create a token.

- Typo: `retires`

- `request configuration object` -> `requests configuration object` (it configures multiple requests)

I would suggest:

```
To do so a token needs to be created and passed through a requests configuration object. Other config such as retries can also be defined there.
```